WordPress/WordPressApi/WordPressOrgXMLRPCApi.swift

```diff
@@ -0,0 +1,180 @@
+import Foundation
+import AFNetworking
+import wpxmlrpc
+
+public class WordPressOrgXMLRPCApi: NSObject
+{
+    public typealias SuccessResponseBlock = (responseObject: AnyObject, httpResponse: NSHTTPURLResponse?) -> ()
+    public typealias FailureReponseBlock = (error: NSError, httpResponse: NSHTTPURLResponse?) -> ()
+
+    private let apiBaseURLString: String
```

Comment:
This is not a "base URL", since we don't build URLs based on it, all XML-RPC requests go to the same URL. I'd make it `private let endpoint: NSURL` instead.